tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ModuleProjectNameListener.java

```diff
@@ -41,13 +41,22 @@
     protected void handleTypedEvent( PropertyContentEvent event )
     {
         updateLocation( op( event ) );
+
+        updateAddJspValidationSupport( op( event ) );
     }
 
     protected NewLiferayModuleProjectOp op( PropertyContentEvent event )
     {
         return event.property().element().nearest( NewLiferayModuleProjectOp.class );
     }
 
+    public static void updateAddJspValidationSupport( final NewLiferayModuleProjectOp op )
+    {
+        final String templateName = op.getProjectTemplateName().content( true );
+
+        op.setAddJspValidationSupport( templateName.equals( "mvc-portlet" ) || templateName.equals( "portlet" ) );
```

Comment:
It is more than just these two project templates that have JSPs right?  many others do as well.  Should we add those as well?

Reply:
Yeah, I will add them